test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -458,6 +458,8 @@ static void alg_test(odp_crypto_op_t op,
 	odp_packet_t pkt = odp_packet_alloc(suite_context.pool,
 					    ref->length + ref->digest_length);
 	CU_ASSERT(pkt != ODP_PACKET_INVALID);
+	if (pkt == ODP_PACKET_INVALID)
+		goto cleanup;
```

Comment:
`CU_ASSERT_FATAL()` is the usual means of dealing with required conditions, but this is OK if that's too abrupt.